account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -0,0 +1,360 @@
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import ormcache
+from odoo.tools.safe_eval import safe_eval
+
+
+class AccountMulticompanyBankWiz(models.TransientModel):
+    _inherit = 'res.partner.bank'
+    _name = 'account.multicompany.bank.wiz'
+    _order = 'id'
+
+    wizard_id = fields.Many2one(
+        comodel_name='account.multicompany.easy.creation.wiz',
+    )
```

Comment:
As we use account.multicompany.easy.creation.wiz model as M2O, we should define ti first.

Reply:
No, that's not needed. And please instead of adding single comments, gather all together in a review.

Reply:
Sure, will add all in single review.
For the M2O, technically we should define the model first. Just My experience regarding parent model being available. I think, order matters.

Reply:
I haven't found any problem with this so far.